python/tvm/topi/x86/utils.py

```diff
@@ -16,8 +16,87 @@
 # under the License.
 """Common x86 related utilities"""
 import tvm
+import tvm._ffi
 
 
+@tvm._ffi.register_func("tvm.topi.x86.utils.target_is_x86")
+def target_is_x86(target):
+    return (
+        target_has_sse41(target)
+        or target_has_sse42(target)
+        or target_has_avx(target)
+        or target_has_avx2(target)
+        or target_has_avx512(target)
+        or target_has_vnni(target)
+        or target
+        in {
+            "k6-2",
+            "x86-64",
+            "atom",
+            "pentium",
+            "nocona",
+            "k8",
+            "i686",
+            "winchip-c6",
+            "prescott",
+            "k6",
+            "athlon-mp",
+            "winchip2",
+            "yonah",
+            "athlon-tbird",
+            "alderlake",
+            "pentium4",
+            "athlon64-sse3",
+            "pentium2",
+            "bonnell",
+            "k8-sse3",
+            "barcelona",
+            "athlon",
+            "c3",
+            "generic",
+            "lakemont",
+            "pentium-mmx",
+            "core2",
+            "pentium3",
+            "btver1",
+            "athlon-fx",
+            "k6-3",
+            "geode",
+            "pentium-m",
+            "i586",
+            "pentium3m",
+            "pentiumpro",
+            "i486",
+            "opteron",
+            "athlon64",
+            "pentium4m",
+            "i386",
+            "opteron-sse3",
+            "amdfam10",
+            "athlon-4",
+            "athlon-xp",
+            "c3-2",
```

Comment:
Do we need the level of details? I don't think people would ever specify these targets... I think sse4.1 - vnni are enough.

Reply:
I agree, sse4.1 looks good. Users can always use requantize_config to change the default behavior.
Done.